test/end-to-end/pages/dashboard/chooseApplication.js

```diff
@@ -32,9 +33,10 @@ module.exports = async function(language ='en') {
     } else {
         console.log('Welsh Dashboard Page...');
         await I.amOnLoadedPage('/dashboard', language);
-        await I.takeScreenshot('./functional-output/dashboard-before.png');
+        await I.saveScreenshot('dashboard-before.png');
```

Comment:
is folder change intentional?

Reply:
yes this is intentional - Codecept has a built in screenshot tool that supports both Puppeteer and Webdriver, and dumps the image to the report folder configured in `sacelabs.conf.js`. So we don't need to define our own function.